go/protocol/chat1/notify.go

```diff
@@ -215,6 +215,11 @@ type ChatTLFFinalizeArg struct {
 	FinalizeInfo ConversationFinalizeInfo `codec:"finalizeInfo" json:"finalizeInfo"`
 }
 
+type ChatTLFResolveArg struct {
+	Uid    keybase1.UID   `codec:"uid" json:"uid"`
+	ConvID ConversationID `codec:"convID" json:"convID"`
```

Comment:
I just don't think this notification has any value unless we pass up the new name, what other information would the UI care about here?